next.clayui.com/content/docs/css/components/button-group.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Button Group"
+description: 'Button groups are used to switch between complementary views for example, but they must never be used for complementary actions, "Change and Cancel" actions, or "Save and Cancel" actions. In those cases, single buttons are the correct solution.'
+---
+
+<div class="nav-toc">
+
+- [Examples](#examples)
+- [Split Button](#split-button)
+- [Button toolbar](#button-toolbar)
+- [Sizes](#sizes)
+- [Vertical variation](#vertical-variation)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/buttons/">Buttons Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+## Examples
+
+<div class="sheet-example">
+	<div class="btn-group" role="group">
+		<button class="btn btn-secondary" type="button">Day</button>
+		<button class="active btn btn-secondary" type="button">Month</button>
+		<button class="btn btn-secondary" type="button">Year</button>
+	</div>
+</div>
+
+```html
+<div class="btn-group" role="group">
+	<button class="btn btn-secondary" type="button">Day</button>
+	<button class="active btn btn-secondary" type="button">Month</button>
+	<button class="btn btn-secondary" type="button">Year</button>
+</div>
+```
+
+## Split Button
+
+<div class="sheet-example">
+	<div class="btn-group" role="group">
```

Comment:
hey @pat270 I had seen in the previous examples that we have `.btn-group.dropdown` and `.btn-group.dropup` but I ended up removing it because I had not seen any effect. What do you think?